src/sass/modules/_m-schemaform.scss

```diff
@@ -161,12 +161,6 @@ label {
   margin-bottom: 7rem;
 }
 
-// Align the intro content of the 22-1990 to match the title
-// Necessary for the 22-1990 and HCA since they don't use schemaform
-.intro-content {
```

Comment:
The 1990 intro page was the only one to use this...and it didn't need it any more.